server/src/test/scala/manifests/queues/SplitsCalculatorSpec.scala

```diff
@@ -113,138 +114,140 @@ class SplitsCalculatorSpec extends CrunchTestLike {
         }
       }
 
-      "Given 0 EEA adults, 4 B5J Nationals and 1 B5J child with a 1.0 adjustment per child" +
+      "Given 0 EEA adults, 4 B5J Nationals and 1 B5J child with a 1.0 adjustment per child" >> {
         "Then I should expect 3 EEA Adults to Desk, 1 B5J child to desk and 1 B5J Adult to eGates" >> {
```

Comment:
is it not one adult per child we are assuming to desk ? here looks like 3 adults to 1 child . or May 1.0 adjustment make it difference ?

Reply:
Good question!. The answer is found in looking at the splits calculator. It's configured to have 50% BJ5JSSK to the egates, and 50% to desks. Thus 50% of 4 is 2, minus one for the child is 1 to the egates, and the rest to the desks.

Looks like the test description is a little wonky too so I'll update it before merging